x-pack/test/api_integration/apis/xpack_main/telemetry/telemetry_local.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from 'expect.js';
+
+const disableCollection = {
+  "persistent":
+  {
+    xpack: {
+      monitoring: {
+        collection: {
+          enabled: false
+        }
+      }
+    }
+  }
+};
+
+export default function ({ getService }) {
+  const supertest = getService('supertest');
+  const esSupertest = getService('esSupertest');
+
+  describe('/api/telemetry/v1/clusters/_stats with monitoring disabled', () => {
+    before('', async () => {
+      await esSupertest.put('/_cluster/settings').send(disableCollection).expect(200);
+    });
+
+    it('should pull local stats', async () => {
+      const timeRange = {
+        min: '2018-07-23T22:07:00Z',
+        max: '2018-07-23T22:13:00Z'
+      };
+
+      const { body } = await supertest
+        .post('/api/telemetry/v1/clusters/_stats')
+        .set('kbn-xsrf', 'xxx')
+        .send({ timeRange })
+        .expect(200);
+
+      expect(body.length).to.be(1);
+      const stats = body[0];
+
+      expect(stats.collection).to.be('local');
```

Comment:
would it be too much to change this to a jest test and use inline snapshots?

Reply:
I haven't seen functional tests using jest yet, but I'd love to see them!

Reply:
oh darn, you're right. I misread the file on this comment

Reply:
I've added 6e0cb9f - I'm curious to hear how terrible that solution is

Reply:
It does what we need it to. I probably would've reached for `expect.to.eql()` using an object, but it's primarily a stylistic difference.

Reply:
Comparing to all of the individual values in this test will be hard. A lot of the `cluster_info` stats won't be the same from one call to another, because it reads live from ES APIs.